e2e/support/commands/api/dashboard.js

```diff
@@ -1,17 +1,23 @@
 Cypress.Commands.add(
   "createDashboard",
-  ({
-    name = "Test Dashboard",
-    enable_embedding,
-    embedding_params,
-    auto_apply_filters,
-    ...dashboardDetails
-  } = {}) => {
+  (
+    {
+      name = "Test Dashboard",
+      enable_embedding,
+      embedding_params,
+      auto_apply_filters,
+      ...dashboardDetails
+    } = {},
+    { wrapId = false, idAlias = "dashboardId" } = {},
+  ) => {
     cy.log(`Create a dashboard: ${name}`);
 
     // For all the possible keys, refer to `src/metabase/api/dashboard.clj`
     cy.request("POST", "/api/dashboard", { name, ...dashboardDetails }).then(
       ({ body }) => {
+        if (wrapId) {
+          cy.wrap(body.id).as(idAlias);
+        }
```

Comment:
Added a change from `master` that's needed in the E2E tests

https://github.com/metabase/metabase/blob/18742132b608d15e3298d8a5dae2377a3f574a66/e2e/support/commands/api/dashboard.js#L18-L20